crates/wasmcloud-control-interface/src/events.rs

```diff
@@ -3,22 +3,27 @@ use serde::{Deserialize, Serialize};
 use std::collections::HashMap;
 
 #[derive(Debug, Serialize, Deserialize, Clone, PartialEq, Default)]
-#[doc(hidden)]
+/// Contains metadata about a control event
 pub struct EventHeader {
+    /// Origin host id
     pub host_origin: String,
+    /// Timestamp in UTC seconds since the epoch
     pub timestamp: u64,
 }
 
 #[derive(Debug, Serialize, Deserialize, Clone, PartialEq)]
-#[doc(hidden)]
+/// A published control event including the event itself and
+/// additional metadata
 pub struct PublishedEvent {
+    /// Event object
     pub event: ControlEvent,
+    /// Event metadata
     pub header: EventHeader,
 }
+
 /// Represents an event that may occur on the lattice control plane. All timestamps
```

Comment:
nitpick, and it's my own fault - but we should say `control interface` rather than `control plane`. I think some kubernetes / CNCF folks feel quite strongly about this.

Reply:
I agree, with the varying definitions of `control plane` it's probably best if we don't mention it at all